libs/search/src/lib/state/reducer.ts

```diff
@@ -10,6 +10,7 @@ export interface SearchState {
   params: {
     filters: SearchFilters
     sortBy?: string
+    resultsLayout?: string
```

Comment:
It's not a search param, actually it has nothing related with the search process in itself.
I really don't know where to put this to be honest.. It should be in application state, but it's true that putting it somewhere in search state allow smart component to react from it.

At least put it at the root but not in params.

Reply:
> It should be in application state,

No because it is related to a search. If you have 2 components making searches in the same page, one may have a layout and the other another one.

It could be considered related to the "_source" - we may link _source with only the fields required for a layout. But that is not really present in the config for now.

Ok moving it to root.